README.md

```diff
@@ -100,6 +100,24 @@ As a JavaScript React component, the project is designed to be used in two diffe
 
    The React component exposes props that can be used to supply data and customise its behaviour. For information about the props, their expected prop-types and default values, see [/src/components/app/index.js](https://github.com/quantumblacklabs/kedro-viz/blob/master/src/components/app/index.js). For examples of the expected data input format, see the mock data examples in [/src/utils/data](https://github.com/quantumblacklabs/kedro-viz/tree/master/src/utils/data), and compare the [resulting demo](https://quantumblacklabs.github.io/kedro-viz/).
 
+## Flags
+
+The following flags are available in the latest release to toggle experimental features:
+
+- `newgraph` - Improved graphing algorithm (default `false`).
```

Comment:
```suggestion
- `newgraph` - From release vX.x. Improved graphing algorithm (defaults to `false`).
```

My thoughts are that it's best to say which release they are introduced from individually as you add them, since "latest" is/will go out of date.

Reply:
Sounds good, I've updated this to include the version.